pkgs/build-support/docker/default.nix

```diff
@@ -209,7 +209,7 @@ rec {
 
       postMount = ''
         echo "Packing raw image..."
-        tar -C mnt --mtime=0 -cf $out .
+        tar -C mnt --mtime='@1' -cf $out .
```

Comment:
I'm not familiar with this notation; I think a comment explaining it would be helpful.